extensions/amp-story/0.1/amp-story.js

```diff
@@ -1078,7 +1078,8 @@ export class AmpStory extends AMP.BaseElement {
    * @return {boolean} True if the screen size matches the desktop media query.
    */
   isDesktop_() {
-    return this.desktopMedia_.matches && !this.platform_.isBot();
+    return !isExperimentOn(this.win, 'disable-amp-story-desktop') &&
```

Comment:
Super nitty: should we put this check last? Since it'll be off most of the time, it doesn't need to run before ``this.desktopMedia_.matches``